infoportal-client/src/features/Database/KoboTable/customization/useCustomColumns.tsx

```diff
@@ -188,7 +186,23 @@ export const useCustomColumns = (): DatatableColumn.Props<KoboMappedAnswer>[] =>
         lastStatusUpdate,
       ]
     }
-
+    const beneficiaries:DatatableColumn.Props<any>[] = [
+      {
+        id: 'beneficiaries',
+        head: m.beneficiaries,
+        type: 'number',
+        renderQuick: (row: KoboAnswerFlat<Protection_gbv.T, any>) => {
+          if (row.new_ben === 'yes') {
+            return row.numb_part || 0;
+          } else if (row.new_ben === 'bno' && row.hh_char_hh_det) {
+            return row.hh_char_hh_det.reduce((count, participant) => {
+              return count + (participant.hh_char_hh_new_ben === 'yes' ? 1 : 0);
+            }, 0);
+          }
+          return 0;
+        }
+      }
+]
```

Comment:
Just minor details, avoid noisy variables when possible. Since it's not reused anywhere it's better to put it inline if the declaration below.
Another minor details, we don't add extra `;` when not necessary.
But good job! 🙂